staging/src/k8s.io/kubectl/pkg/cmd/get/get_test.go

```diff
@@ -2717,7 +2768,14 @@ func podTableObjBody(codec runtime.Codec, pods ...corev1.Pod) io.ReadCloser {
 			Cells:  []interface{}{pods[i].Name, "0/0", "", int64(0), "<unknown>", "<none>", "<none>", "<none>", "<none>"},
 		})
 	}
-	return cmdtesting.ObjBody(codec, table)
+	data, err := json.Marshal(table)
+	if err != nil {
+		panic(err)
+	}
+	if !strings.Contains(string(data), `"meta.k8s.io/v1"`) {
+		panic("expected v1, got " + string(data))
+	}
+	return cmdtesting.BytesBody(data)
```

Comment:
this ensures we're serializing the data we expected (the type aliases were throwing off the codec's preferred external version)